STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.Mappings.Headers.cs

```diff
@@ -0,0 +1,401 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Net.Http.Headers;
+using STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges;
+
+namespace STX.REST.RESTFulSense.Clients.Services.Foundations.HttpExchanges
+{
+    internal partial class HttpExchangeService
+    {
+        private static void MapToHttpRequestHeaders(
+            HttpExchangeRequestHeaders httpExchangeRequestHeaders,
+            HttpRequestHeaders httpRequestHeaders)
+        {
+            if (httpExchangeRequestHeaders is null)
+                return;
+
+            if (httpExchangeRequestHeaders.Accept != null)
```

Comment:
```suggestion
            if (httpExchangeRequestHeaders.Accept is not null)
```
Do the same for all the others.

Reply:
Fixed

Reply:
Fixed